adapters/Vungle/VungleAdapter/GADMAdapterVungleDelegate.h

```diff
@@ -35,13 +33,20 @@ typedef NS_ENUM(NSUInteger, BannerRouterDelegateState) {
 - (void)adAvailable;
 - (void)adNotAvailable:(nonnull NSError *)error;
 - (void)willShowAd;
-- (void)willCloseAd:(BOOL)completedView didDownload:(BOOL)didDownload;
-- (void)didCloseAd:(BOOL)completedView didDownload:(BOOL)didDownload;
+- (void)willCloseAd;
+- (void)didCloseAd;
+- (void)trackClick;
 
 @optional
 
-// Get banner request object
-@property(nonatomic, nonnull) GADMAdapterVungleBannerRequest *bannerRequest;
+// Differentiate two banners with same placement ID
+@property(nonatomic, nullable) NSString *uniquePubRequestID;
```

Comment:
(nonatomic, copy, nullable)

Make it be a copy instead of being a strong reference, then other code calling this doesn't need to explicitly call copy when referring to this property.

Reply:
@ericleich Hi Eric, I'm sorry for the late response. I took PTO yesterday. Thanks a lot for the guidance, I will add 'copy' attribute.  :)

Reply:
@ericleich Added, thanks. ^_^